php/libraries/Module.class.inc

```diff
@@ -58,6 +58,30 @@ abstract class Module extends \LORIS\Router\PrefixRouter
         );
     }
 
+    /**
+     * Retrieve all active module descriptors from the given database, indexed
+     * by their primary key in the database.
+     *
+     * @param \Database $db an open connection to a database containing a 'modules'
+     *                      table
+     *
+     * @return \Module[]
+     */
+    static public function getActiveModulesIndexed(\Database $db): array
```

Comment:
I think this function belongs in a LORIS instance object more then this class...just saying

Reply:
There is no LORIS instance object.

Reply:
create one

Reply:
That is not within the scope of this PR

Reply:
#6022